DoggyFriction/Services/DebtService.cs

```diff
@@ -25,7 +25,7 @@ public IEnumerable<Debt> GetDebts(IEnumerable<Action> actions)
                                 if (!debtContainers.ContainsKey(key)) {
```

Comment:
Let's extract this `if` and latter `debtContainer[key]` to some extension, or refactor it some other way